Dockerfile

```diff
@@ -28,4 +33,5 @@ COPY --from=jdtls-download /jdtls /jdtls/
 COPY --from=addon-build /root/.m2/repository/io/konveyor/tackle/java-analyzer-bundle.core/1.0.0-SNAPSHOT/java-analyzer-bundle.core-1.0.0-SNAPSHOT.jar /jdtls/java-analyzer-bundle/java-analyzer-bundle.core/target/
 COPY --from=fernflower /output/fernflower.jar /bin/fernflower.jar
 COPY --from=addon-build /app/hack/lsp-cli /bin/lsp-cli
+COPY --from=maven-index /maven.default.index /maven.default.index
```

Comment:
do we want this at the root of the container or should it be somewhere else?

Reply:
@shawn-hurley couldn't think of any other location, would `/usr/local/etc` be ok?